### ISSUE TITLE
[Broken Website] tomshardware.com

### ISSUE THREAD

**Issue by The-Solutor** [darkreader/darkreader#12480]
### Prerequisites

- [X] I [searched for any existing report](https://github.com/darkreader/darkreader/issues?q=is%3Aissue) about this website issue to avoid opening a duplicate.
- [X] I can reproduce this website issue in a new, unmodified web browser profile with Dark Reader installed as the only extension.

### Website Issue Description

Since few days DR stopped working correctly on tomshardware, and only the borders are dark

### Website Address

tomshardware.com

### Steps To Reproduce

Nothing special

Just install darkreader (no matter if on chrome based or FF based browser) then open tomshardware,com

### Screenshots

![immagine](https://github.com/darkreader/darkreader/assets/538364/9b03f118-dfb3-4fed-ac43-7055dbb8e37b)

### Operating System

Windows 10 (no matter the release)

### Web Browser name and version

Tested on current Vivaldi, Waterfox and others

### Dark Reader version

4.9.80

**Comment by alexanderby** [comment]
Fixed already by https://github.com/darkreader/darkreader/commit/ed819d6153638a962cf2d9a46fc76ef6541d0351 and https://github.com/darkreader/darkreader/commit/59a755f69857b294cfc78df27973885c8ab4e8c8 and will be released next week.

**Comment by The-Solutor** [comment]
Thanks.

**Comment by NightClawZ** [comment]
Hello,

The issues seems similar to what I have for pcgamer.com and gamesradar.com, also with Dark Reader version 4.9.80. 
I suppose there is a great chance that those websites will also be fixed next week right ? :)
Thank you for you answer and your work. 

![image](https://github.com/darkreader/darkreader/assets/4301769/0fdcccf9-a581-4023-a2f9-f699d9ed792d)
![image](https://github.com/darkreader/darkreader/assets/4301769/f4808ede-fd39-4759-a34d-b48a0927c2e2)

**Comment by erkinalp** [comment]
Can you test #12495 ?

**Comment by MultiKoopa** [comment]
Same issue here, want to add windowscentral.com as well

Not sure how I'd test #12495

**Comment by erkinalp** [comment]
@MultiKoopa open developer tools, apply the change in the PR

**Comment by NightClawZ** [comment]
@erkinalp 
Hello, 
Not OP but globally it works, except the menu at the top : 
![image](https://github.com/darkreader/darkreader/assets/4301769/043ddea3-d8dc-45ac-9dfa-c6dec88a62d5)

**Comment by hex-developer** [comment]
This is my progress:
```
INVERT
img[src="/styles/tomshardware/tomshardware/toms-hardware-logo.png"]
div.trophyShowcase.trophyShowcase--postbit

CSS
.slice-container-trending {
    --trending-background-color: var(--darkreader-neutral-background) !important;
}
.legacy-container#main,
.burgerbar {
    background-color: var(--darkreader-neutral-background) !important;
}
.top-featured .feature-block-item-wrapper,
.top-featured .feature-block-item,
.listingResult {
    background-color: rgb(18, 18, 18) !important;
}
h3,
article div.content {
    background-color: black !important;
}
a.list-heading-link,
.trending__label,
.locale-selector__text,
.locale-selector__icon--chevron-down,
.button-search,
.menu-item a,
p.synopsis {
    color: #ededed !important;
}
```

**Comment by hex-developer** [comment]
The number of CSS rules that need to be overridden is quite large and I will not be spending any more of my time for this. This issue should probably be closed, as #12507 can help create a generic fix for other future-brand websites.

**Comment by alexanderby** [comment]
Should be fixed in version 4.9.81.
